Assets/Scripts/Actors/BuddyStats.cs

```diff
@@ -3,18 +3,29 @@
 
 public class BuddyStats : MonoBehaviour
 {
-	[SerializeField] Stat statType = Stat.Invalid;
-	[SerializeField] bool canDecreaseStamina = true;
-	[SerializeField] float decreaseResourcesTime = 60.0f; //seconds
-	[SerializeField] float currResourceTimer = 0.0f;
+	public Stat statType = Stat.Invalid;
 	[SerializeField] int startingApples = 10;
 	[SerializeField] int apples = 0;
-
-	public GodTag owner = null;
+	[SerializeField] GodTag _startingOwner = null;
 
 	PlayerActorStats _actorStats = null;
```

Comment:
This is poorly named, it should be something like `_ownerStats`. As it is it implies that it refers to the buddy's `PlayerActorStats` component.